tracer/test/Datadog.Trace.Security.IntegrationTests/AspNetCore2RateLimiter.cs

```diff
@@ -0,0 +1,41 @@
+// <copyright file="AspNetCore2RateLimiter.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if NETCOREAPP2_1
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Datadog.Trace.TestHelpers;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.Security.IntegrationTests
+{
+    public class AspNetCore2RateLimiter : AspNetBase, IDisposable
+    {
+        public AspNetCore2RateLimiter(ITestOutputHelper outputHelper)
+            : base("AspNetCore2", outputHelper, "/shutdown")
+        {
+        }
+
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        [Trait("RunOnWindows", "True")]
+        [Trait("Category", "ArmUnsupported")]
+        public async Task TestRateLimiterSecurity(bool enableSecurity, string url = DefaultAttackUrl)
```

Comment:
Can you also set up a test where the number of traces does not exceed the rate limit? That way we can assert that all of the traces are kept

Reply:
I added this in another branch, where I've been trying to make the test more stable: 1073823f736c82da5c6b4a22c5ed663c9f73fe8f

Once it passes CI, I'll bring it into this branch.